src/Thumbnail.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classSet from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+
+const Thumbnail = React.createClass({
+	mixins: [BootstrapMixin],
+
+	getDefaultProps() {
+		return {
+			bsClass: 'thumbnail'
+		};
+	},
+
+	render() {
+		let classes = this.getBsClassSet();
+
+		return (
+			<a {...this.props} className={classSet(this.props.className, classes)}>
```

Comment:
Sounds like a great way of differentiating, I will get to work on updating this - thank you!